src/software/test_util/test_util.cpp

```diff
@@ -110,4 +110,15 @@ namespace TestUtil
         }
         return states;
     }
+
+    void handleTestRobotLog(TbotsProto_RobotLog robot_log)
+    {
+        LOG(INFO) << "[TEST ROBOT " << robot_log.robot_id << "]["
+                  << TbotsProto::LogLevel_Name(
+                         static_cast<TbotsProto::LogLevel>(robot_log.log_level))
+                  << "]"
+                  << "[" << robot_log.file_name << ":" << robot_log.line_number
+                  << "]: " << robot_log.log_msg;
+    }
+
```

Comment:
I don't think this is necessary

Reply:
I believe so

Reply:
can you remove `handleTestRobotLog` then please?

Reply:
done